rcode-reader/src/main/java/eu/rssw/pct/TypeInfo.java

```diff
@@ -73,7 +73,25 @@ public boolean hasTempTable(String inName) {
     return false;
   }
 
+  public boolean hasProperty(String name) {
+    for (PropertyElement prop : properties) {
+      if (prop.getName().equalsIgnoreCase(name) && (prop.isPublic() || prop.isProtected()))
+        return true;
+    }
+    return false;
+  }
+
+  protected PropertyElement getProperty(String name) {
+    // Only for testing
+    for (PropertyElement prop : properties) {
+      if (prop.getName().equalsIgnoreCase(name))
+        return prop;
+    }
+    return null;
+  }
+
   public boolean hasBuffer(String inName) {
+    // TODO Can it be abbreviated ??
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)